smartseq2_single_sample/tasks/htseq.wdl

```diff
@@ -0,0 +1,24 @@
+task htseq_count {
+  File aligned_bam
+  File gtf 
+  String featuretype
+  String output_filename
+  command {
+    htseq-count --format=bam --stranded=no \
+      --order=pos \
+      --type="${featuretype}" \
+      --idattr="${featuretype}_id" \
+      "${aligned_bam}" \
+      "${gtf}" \
+      > "${output_filename}.${featuretype}.htseq.count.txt"
+    }
+  runtime {
+    docker:"humancellatlas/python3-scientific:0.1.1"
```

Comment:
Just curious, what the difference between 0.1.1, 0.1.2, 0.1.3? I remember we only had 0.1.0 before for this image.

Reply:
I am not sure the others. I only added htseq package to 0.1.0 and made a docker with tag 0.1.1

Reply:
Yep. 0.1.3 contains sctools 0.1.4

Reply:
@ambrosejcarr you mean 0.1.2 contains sctools 0.1.2? If so, it is reasonable and I would update my docker tracking table. Thanks!

Reply:
No, because multiple things update python3-scientific, it will not necessarily track sctools' version number (which must be incremented to test, due to pypi requirements).

Reply:
I see, thanks!